Configurations/unix-Makefile.tmpl

```diff
@@ -657,7 +657,8 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl $(ERROR_REBUILD) -internal )
 	( cd $(SRCDIR)/engines; \
           for E in *.ec ; do \
-              $(PERL) ../util/mkerr.pl -conf $$E `basename $$E .ec`.c ; \
+              $(PERL) ../util/mkerr.pl $(ERROR_REBUILD) \
+                -conf $$E `basename $$E .ec`.c ; \
```

Comment:
oops.  restored the -static and also fixed the linewrap length.  additional commit pushed.